blockchain/node/src/chain_spec.rs

```diff
@@ -76,7 +84,7 @@ pub fn development_config() -> Result<ChainSpec, String> {
         // Protocol ID
         None,
         // Properties
-        None,
+        Some(props),
```

Comment:
We shouldn't need a variable for this, just `Some(fractal_properties())` is good. Same in mainnet_config.